modules/ROOT/pages/design-publish.adoc

```diff
@@ -14,6 +14,17 @@ image:apid-example-page-in-exchange.png[An example page for a specification publ
 <2> The middle pane, which describes the specification. You write the description in Exchange after publishing the specification.
 <3> The right pane, which displays metadata about the specification.
 
+== Before You Begin
+
+It is a good practice to remove unreferenced files before publishing a project to Anypoint Exchange. A unreferenced file is a file that meets both of these conditions:
+
+* It is not referenced from a project's root file.
+
+* It is not referenced from any file that the root file references.
```

Comment:
re-thinking about this....maybe it is clearer to say "referenced directly or indirectly from the root file"...otherwise is sounds like you can reference it directly or from the level one referenced files, but not from a reference of a reference